layouts/default.vue

```diff
@@ -260,7 +292,13 @@ export default {
         display: flex;
         margin-left: auto;
         align-items: center;
-        margin-right: 1rem;
+        margin-right: 0.5rem;
+
+        .theme-toggle {
+          @media screen and (max-width: 375px) {
+            display: none;
+          }
```

Comment:
If I'm reading this right, you're hiding the theme toggle if the screen is smaller than 375px? Doesn't that defeat the whole purpose of the button, to be able to switch modes from mobile?

Reply:
It does remove the button, which is the easiest way to make the page fit on mobile phones that small. I was using the logic, "If a phone is that small, then lets hope they have a system wide dark mode toggle like an iPhone 5s." But, that might not be true for other older, smaller phones. Also, maybe a user only wants Modrinth to be in dark mode.

Possible solutions?
- Shrink logo
- Get rid of "Modrinth" wording in logo and just use icon
- Shrink buttons (imo not that great of a solution)

Also, semi-unrelated but, wouldn't it be nice to have the mobile navigation on the bottom of the phone?

Reply:
The second image in #203 is at 360px.

Reply:
How did you get that image? 360px looks like this for me (using the changed you suggested):
![image](https://user-images.githubusercontent.com/44736536/117606663-31057880-b10f-11eb-9a0a-7da5058ba7c6.png)

Reply:
That doesn't *look* like 0.5rem to me, but I could be wrong; the way I did it was adding `style="margin-[left or right]:-0.5rem"` to one of the HTML tags lol  
In that case, switching from logo with text to just logo icon makes the most sense under 375px imo

Reply:
Everything should be fixed on 8a0a698aa3feca1e8f7ec33bb50aa042de6eb8b3. *This time no dark mode buttons were harmed in the process.*